### DISCUSSION TITLE
Suggestion for Rules Directed at PowerShell

### DISCUSSION THREAD

**Discussion by austinsonger** [SigmaHQ/sigma#2472] [Ideas]
**Change all rule file names like the following:**

`powershell_clear_powershell_history.yml`

**to** 

`posh_clear_powershell_history.yml`

**Comment by frack113** [comment]
HI,
For my rule I have add `_ps_` or `_pm_`  to replace "_in_scriptblocktext"," _in_contextinfo",..
So That give me
- "powershell_ps_suspicious_ad_group_reco" -> **"posh_ps_suspicious_ad_group_reco"**
- "powershell_pm_suspicious_ad_group_reco" -> **"posh_pm_suspicious_ad_group_reco"**

**Reply by austinsonger** [reply]
Yeah I like that addition too. I just feel like these little changes would help the organizing of PowerShell rules as well as gaining some context of the rule.
